components/renku_data_services/project/orm.py

```diff
@@ -55,26 +61,17 @@ def dump(self) -> models.Project:
             visibility=self.visibility,
             created_by=models.Member(id=self.created_by_id),
             creation_date=self.creation_date,
-            repositories=self.repositories,
+            repositories=[models.Repository(r.url) for r in self.repositories],
             description=self.description,
         )
 
 
-class ProjectMemberORM(BaseORM):
-    """A Renku native project."""
-
-    __tablename__ = "projects_members"
+class ProjectRepositoryORM(BaseORM):
+    """Renku project repositories."""
 
-    project_id: Mapped[str] = mapped_column("project_id", String(26), primary_key=True)
-    role: Mapped[Role]
-    # NOTE: This is KeyCloak ID's of a user and not a unique ID
-    id: Mapped[str] = mapped_column("id", String(36), unique=False, index=True, primary_key=True)
-
-    @classmethod
-    def load(cls, member: models.MemberWithRole, project_id: str):
-        """Create an instance from MemberWithRole."""
-        return cls(project_id=project_id, role=member.role, id=member.member.id)
+    __tablename__ = "projects_repositories"
 
-    def dump(self) -> models.MemberWithRole:
-        """Create a project members model."""
-        return models.MemberWithRole(member=models.Member(id=self.id), role=self.role)
+    id: Mapped[int] = mapped_column("id", Integer, primary_key=True, default=None, init=False)
+    url: Mapped[str] = mapped_column("url", String(99))
```

Comment:
I think this is a bit too short. I dont want to have to fail people for using a longer than expected git url. Searching around on google seems to indicate that urls longer than 2000 characters stop working on most browsers. So going with a higher number like that is probably a good idea.